test/import-test.js

```diff
@@ -312,7 +312,7 @@ describe('#importDataElement', () => {
     expect(entry.value.constraints).to.have.length(1);
     expect(entry.value.constraints[0]).to.be.instanceof(CodeConstraint);
   console.log("Test 20: entry.value.constraints[0] = "+ JSON.stringify(entry.value.constraints[0]));  // missing path
-    expect(entry.value.constraints[0].path).to.eql([id('codeConstraintOnValueChildOut','CodedFromValueSet')]);   // fails here because the path is missing
+    expect(entry.value.constraints[0].path).to.eql([id('primitive','concept')]);   // fails here because the path is missing
```

Comment:
Note: I think there is a `pid` function for primitives.  E.g., `pid('concept')`.

Reply:
Also, I think you can remove the comment about it failing.